autoload/magit/mapping.vim

```diff
@@ -2,6 +2,8 @@
 let g:magit_stage_file_mapping     = get(g:, 'magit_stage_file_mapping',        'F' )
 let g:magit_stage_hunk_mapping     = get(g:, 'magit_stage_hunk_mapping',        'S' )
 let g:magit_stage_line_mapping     = get(g:, 'magit_stage_line_mapping',        'L' )
+let g:magit_stage_all_mapping      = get(g:, 'magit_stage_all_mapping',         'gA' )
+let g:magit_unstage_all_mapping    = get(g:, 'magit_unstage_all_mapping',       'gU' )
```

Comment:
I'm using `gA` and `gU`, but they can be obviously changed (gS/gR, for example). `gR` could be used in the future for remotes, so I wouldn't use it.